Sources/MarkupRenderer.swift

```diff
@@ -19,10 +19,21 @@ public final class MarkupRenderer {
 		self.baseFont = baseFont
 	}
 
-	public func render(text: String) -> NSAttributedString {
-		let elements = MarkupParser.parse(text: text)
-		let attributes = [NSAttributedString.Key.font: baseFont]
+	public func render(text: String, foregroundColor: UIColor? = nil, backgroundColor: UIColor? = nil) -> NSAttributedString {
```

Comment:
Why do you limit this specifically to foregroundColor and backgroundColor?
Why not do this:
```
public func render(text: String, withAttributes attributes: [NSAttributedString.Key: Any] = []) -> NSAttributedString {
	...
	return return elements.map { $0.render(withAttributes: [NSAttributedString.Key.font: baseFont] + attributes) }.joined()
}
```

Reply:
done